### DISCUSSION TITLE
[视频] Open Source Software Licensing Basics for Corporate Users

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#61] [资料分享]
[Heather Meeker](https://www.amazon.com/Heather-J-Meeker/e/B001JSAT12/ref=dp_byline_cont_pop_book_1) 的系列视频，推荐~

This presentation is suitable for lawyers, business persons or engineers who want to understand open source software licensing and how to comply with open source licenses in a corporate setting.

https://www.youtube.com/watch?v=gF4b1TA5Q5w&list=PLAVikl6VpxPeBtplWOnfzNmiUz529AYAy

**Comment by wangleo61** [comment]
Good sharing and very useful

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: Will Wang ***@***.***>
发送时间: Monday, March 21, 2022 10:46:21 AM
收件人: kaiyuanshe/ONES ***@***.***>
抄送: Subscribed ***@***.***>
主题: [kaiyuanshe/ONES] [视频] Open Source Software Licensing Basics for Corporate Users (Discussion #61)


This presentation is suitable for lawyers, business persons or engineers who want to understand open source software licensing and how to comply with open source licenses in a corporate setting.

https://www.youtube.com/watch?v=gF4b1TA5Q5w&list=PLAVikl6VpxPeBtplWOnfzNmiUz529AYAy

―
Reply to this email directly, view it on GitHub<https://github.com/kaiyuanshe/ONES/discussions/61>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AO5RA33TKIYF4KKMFWDKHFLVA7PH3ANCNFSM5RGNKM6A>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
